doc/doc-site/docs/resources/integrators/api/trending.md

```diff
@@ -0,0 +1,80 @@
+---
+slug: /integrators/api/trending
+title: 📱 Trending
+hide_title: false
+description: Trending API
+---
+
+# The trending API
+
+Route: `/api/trending`.
+
+Query the endpoint with no query params for retrieving the top 25 trending
+markets.
+
+Prices are expressed in terms of the base asset being the emojicoin.
+  - `"quote_price": 0.0123` indicates `1` emojicoin is worth `0.0123` APT.
+  - `"usd_price": 0.20` indicates `1` emojicoin is worth `0.20 USD`.
+
+**NOTE: `market_cap_usd` and `usd_price` may be absent if APT/USD isn't fetched successfully.**
+
+All APT and emojicoin values are returned in their decimalized formats; that
+is, they are divided by $10^8$.
+
+All addresses are [AIP-40] compliant, meaning that they include leading 0s.
+
+`base` always refers to the emojicoin.
+
+`quote` always refers to APT.
+
+`theoretical_curve_price` is the current spot price.
+
+### Example return data
+
+```json
```

Comment:
- [ ] Escape the markdown lint line length just before and just after this block